src/Tribe/Admin/Tickets_Settings.php

```diff
@@ -1,10 +1,14 @@
 <?php
+namespace Tribe\Tickets\Admin;
 
 use Tribe\Admin\Troubleshooting as Troubleshooting;
+use Tribe__Settings;
+use Tribe__Settings_Tab;
+
 /**
  * Manages the admin settings UI in relation to ticket configuration.
  */
-class Tribe__Tickets__Admin__Ticket_Settings {
+class Tickets_Settings {
```

Comment:
This should be `Ticket_Settings` - See my comment [here](https://github.com/the-events-calendar/the-events-calendar/pull/3813/files#r849440547) for a bit more info.

Reply:
Revising based on our conversation in Slack and the resulting update to TEC - change to `src/Tribe/Admin/Settings.php`